requirements.txt

```diff
@@ -28,3 +28,4 @@ xarray==0.19.0
 zipp
 beautifulsoup4
 ratelimit
+xlrd
```

Comment:
This file has some conflicts, please take the latest pull from the master branch and send for review.
[
![image](https://github.com/user-attachments/assets/512cbd0f-4c15-4d88-bad8-54be25bf8dfd)](https://screenshot.googleplex.com/BhaR3Zw9VRWrwSW)